### ISSUE TITLE
Why are 'w2r' and 'w2g' not in the loss? One guess is that this code was originally used for quadbayer remosaic?

### ISSUE THREAD

**Issue by quhaoooo** [viengiaan/RGBW_Remosaicing#2]

**Comment by viengiaan** [comment]
Note that we did not use mapping loss in this work.

**Comment by quhaoooo** [comment]
OK, If I use this method to accomplish quadbayer remosaicing, what modifications do you suggest I make?

**Comment by viengiaan** [comment]
For the loss function in this work, you can directly use it. 

However, there is one thing you should note: since there are loss functions in the color domain (RGB), we use the "simple_ISP_for_training" function to transform (GRBG) Bayer images into color images. 

Thus, I think you should modify a Bayer pattern according to your work in the "simple_ISP_for_training" function.

**Comment by quhaoooo** [comment]
Thank you for your generous share !

**Comment by quhaoooo** [comment]
I changed GRBG to RGGB for the quadbayer and it worked. But I want to ask if there is any modification in network structure? it seems that LKI is generic. Another issue is that the saved images are not being updated during the training process. What could be the problem?
